### DISCUSSION TITLE
Limitations et Utilisations API

### DISCUSSION THREAD

**Discussion by Elivrio** [bokub/conso-api#72] [Général]
Bonjour, 

Pour mon entreprise, je suis en train d'étudier l'intégration de cet API au sein de notre application.

Après une installation sur un gros site possédant potentiellement plusieurs compteurs LPR, nous sommes inquiet tant qu'aux limitations dû au nombre de requête. 

En effet, si un site possède disons 15 compteurs et qu'il souhaite faire un relevé quotidien de toutes les requêtes possible, cela fait entre 45 et 75 requêtes par jour (en fonction de la présence de production ou non), potentiellement plus si certaines requêtes échouent (pour une raison quelconque) et que les opérateurs décident de relancer des requêtes manuellement. 

J'imagine un scénario "catastrophe" bien sûr, mais je préfère prévenir que guérir
Ma question est donc simplement, à partir de combien de requête une IP est-elle considérée comme un spam et en danger de ban?

Merci d'avance et Bonne journée

**Comment by bokub** [comment]
Hello @Elivrio 

Tu as tout à fait raison d'avoir posé la question !

Si tu as une entreprise, tu peux contractualiser directement avec Enedis et appeler leurs API de la même manière que Conso API les appelle (mon code est open-source donc tu peux t'en inspirer).

Tu auras ainsi de gros quotas juste pour toi sans risque que ton IP se fasse bannir par mon système automatisé.
